content/releases.md

```diff
@@ -1,4 +1,89 @@
+- **[Helm chart](https://github.com/eclipse-iofog/helm/releases/tag/v1.2.0)** for easy installation of ioFog Edge Compute Network to existing Kubernetes cluster (see [helm tutorial](/docs/1.2.0/getting-started/how-to-helm.html))
+- **[Platform tools](https://github.com/eclipse-iofog/platform/tree/1.2.0)** for easy infrastructure and Kubernetes cluster setup on Google Cloud Platform and Packet (see [platform tutorial](/docs/1.2.0/tools/platform-tools.html))
+- Many bugfixes in the ioFog engine: [controller](https://github.com/eclipse-iofog/Controller/releases/tag/v1.2.0), [connector](https://github.com/eclipse-iofog/Connector/releases/tag/v1.2.0) and [agent](https://github.com/eclipse-iofog/Agent/releases/tag/v1.2.0)
+
+### Changelogs
+
+##### Agent (v1.2.0)
+
+* Send Agent's external IP to Controller
+* Bugfix: Selecting the network interface that has Controller connectivity
+* Bugfix: Use local docker images when offline
+
+##### Connector (v1.2.0)
+
+* Limit port range to 50 ports in default configuration file
+
+##### Controller (v1.2.0)
+
+* Return Agent's external IP for Kubelet
+* Add uptime to status endpoint
+* Bugfix: Requests not failing if with additional properties
+
+##### Kubelet (v1.2.0)
+
+* Initial release!
+
+##### Iofogctl (v1.2.0)
+
+* Initial release!
+
+##### Helm Chart (v1.2.0)
+
+* Initial release!
+
+##### Platform Tools (v1.2.0)
+
+* Initial release!
+
+##### Quickstart + Tutorial (v1.2.0)
+
+* 
+
+### Known Issues
+
+- Smart Camera Docker Container sometimes hangs after start ups.
+- ioFog Agent local API for receiving ioMessages from microservices will crap out after about 700 messages
+- Controller rejects requests that have unknown properties
+- Agent gives internal IP to Controller instead of public IP when on GCP
+- Agent not using local images when no internet connection
+- "iofog-agent info" hangs when multiple network adapters and no internet connection
+- Agent won't start when no internet connection
+- Agent will not start docker containers when Internet connection is not available
+- iofog-agent info command hangs and never returns information when Internet connection not available
+- ioFog Agent will not start when GPS mode is "auto" and Internet connection is not available
+- SQLite error on controller endpoint /api/v3/signup
+- Demo scripts in develop branch fail on macos with getopt error.
+- VideoProcessing doesn't succesfully load cameraSource
+- Master branch of demo repo tests fail sometimes on Pipelines
+- ARM TCP Docker issue causing images to not spawn
+- Route creation using iofog-controller cli failed, then succeeded with no change to the cli command
+- Update Controller npm dependencies
+- Update nodejs-sdk npm dependencies
+- Concurrent Requests to Provision an Agent on Controller API fail
+- New Connector has no authentication enabled
+- Need to change "publicMode": true to "publicMode": false in example for microservice "JSON ADD File Schema"
```

Comment:
This is done, @baghbidi Is this fixed overall, so publicMode: true works properly again?